discojs/package.json

```diff
@@ -17,8 +17,10 @@
     "@koush/wrtc": "0.5",
     "@tensorflow/tfjs": "4",
     "@tensorflow/tfjs-node": "4",
+    "@types/d3": "^7.4.0",
     "@types/msgpack-lite": "0.1",
     "axios": "0.27",
+    "d3": "^7.6.1",
```

Comment:
following my code review, it's actually relevant that you added d3 to `discojs`'s dependencies, although your current code would only require the dependency within `web-client`